src/main/kotlin/no/nav/syfo/application/VedtakService.kt

```diff
@@ -14,6 +14,7 @@ class VedtakService(
     private val journalforingService: IJournalforingService,
     private val infotrygdService: InfotrygdService,
     private val esyfovarselHendelseProducer: IEsyfovarselHendelseProducer,
+    private val vedtakFattetProducer: IVedtakFattetProducer,
```

Comment:
Kan vurdere å ha en `IVedtakProducer` som tar inn disse to og tar seg av både esyfovarsel- og vedtakfattet-produsering, litt sånn ala vi har det i isarbeidsuforhet.